react_app/src/components/ProfileButton.js

```diff
@@ -26,10 +38,21 @@ class ProfileButton extends React.Component {
   }
 
   doLogOut() {
+    /**
+     * scattering fetches like this around your components won't scale.
+     * what you want to do is look to creating a "services" folder and
+     * put them in there, so this code would then just be;
+     *
+     *   doLogOut() {
+     *      ctfService.logout()
+     *   }
+     *
+     */
```

Comment:
Good plan, I can probably do the same for my graphql queries to keep all the network stuff together